frontend/packages/kubevirt-plugin/src/components/modals/nic-modal/nic-modal.tsx

```diff
@@ -253,7 +263,11 @@ export const NICModal = withHandlePromise((props: NICModalProps) => {
             network={resultNetwork}
             onChange={onNetworkChoiceChange}
             nads={nads}
-            allowPodNetwork={allowPodNetwork}
+            allowPodNetwork={
+              editConfig?.allowPodNetworkOverride != null
```

Comment:
does `editConfig?.allowPodNetworkOverride || allowPodNetwork` makes sense here ?

Reply:
what do you mean by "make sense"? 
Pod network is allowed only for one nic in kubevirt, so we have to disable it here for duplicate networks.

Reply:
> what do you mean by "make sense"?
`a != null ? a : b`
can be `a || b`